ironfish-cli/src/commands/wallet/export.ts

```diff
@@ -104,7 +116,7 @@ export class ExportCommand extends IronfishCommand {
       return
     }
 
-    if (color && !flags.language && !flags.mnemonic) {
+    if (color && flags.json) {
```

Comment:
I think this is broken, if you provide the `--json` flag it will try to output the `bech32m` output as json.